hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ReplicationManager.java

```diff
@@ -263,12 +263,13 @@ private synchronized void run() {
         wait(rmConf.getInterval());
       }
     } catch (Throwable t) {
-      // When we get runtime exception, we should terminate SCM.
-      LOG.error("Exception in Replication Monitor Thread.", t);
       if (t instanceof InterruptedException) {
-        Thread.currentThread().interrupt();
```

Comment:
Even though we know the thread is exiting, I think you are still supposed to keep the line `Thread.currentThread().interrupt();` incase something else needs it before the thread exits.